### ISSUE TITLE
Verify IETF nits are resolved

### ISSUE THREAD

**Issue by nickl-** [json-schema/json-schema#2]
At http://tools.ietf.org/idnits?url=http://tools.ietf.org/id/draft-zyp-json-schema-03.txt the following problems were reported for I-D(3) or draft 3 which should be resolved going forward.

idnits 2.12.13 

tmp/draft-zyp-json-schema-03.txt:

  Checking boilerplate required by RFC 5378 and the IETF Trust (see
  http://trustee.ietf.org/license-info):

---

```
 No issues found here.
```

  Checking nits according to http://www.ietf.org/id-info/1id-guidelines.txt:

---

```
 No issues found here.
```

  Checking nits according to http://www.ietf.org/id-info/checklist :

---

```
 No issues found here.
```

  Miscellaneous warnings:

---
- The copyright year in the IETF Trust and authors Copyright Line does not
   match the current year
- The document date (November 22, 2010) is 638 days in the past.  Is this
   intentional?

Checking references for intended status: Informational:

---
- Unused Reference: 'RFC3339' is defined on line 1073, but no explicit
   reference was found in the text
- Unused Reference: 'RFC2616' is defined on line 1088, but no explicit
   reference was found in the text
- Unused Reference: 'I-D.hammer-discovery' is defined on line 1104, but no
   explicit reference was found in the text
- Unused Reference: 'I-D.gregorio-uritemplate' is defined on line 1110,
   but no explicit reference was found in the text
- Unused Reference: 'I-D.nottingham-http-link-header' is defined on line
   1116, but no explicit reference was found in the text
- Obsolete normative reference: RFC 2396 (Obsoleted by RFC 3986)
- Outdated reference: draft-gregorio-uritemplate has been published as RFC
   6570
- Outdated reference: draft-nottingham-http-link-header has been published
   as RFC 5988

```
     Summary: 1 error (**), 8 warnings (==), 1 comment (--).

     Run idnits with the --verbose option for more detailed information about
     the items above.
```
